### PR TITLE
zh-trans: zh/docs/reference/glossary/flexvolume.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: zh/docs/reference/glossary/flexvolume.md

### CODE REVIEW

File: content/zh/docs/reference/glossary/flexvolume.md [hunk 1]
```diff
@@ -11,12 +26,26 @@ aka:
...
 
 <!--more--> 
 
+<!--
 Flexvolumes enable users to write their own drivers and add support for their volumes in Kubernetes. FlexVolume driver binaries and dependencies must be installed on host machines. This requires root access. The Storage SIG suggests implementing a {{< glossary_tooltip text="CSI" term_id="csi" >}} driver if possible since it addresses the limitations with Flexvolumes.
+-->
 
+Flexvolume 允许用户编写自己的驱动程序，并在 Kubernetes 中添加对卷的支持。
+FlexVolume 驱动程序的二进制文件和依赖项必须安装在主机上。这需要 root 权限。
+如果可能的话，Storage SIG 建议实现 {{< glossary_tooltip text="CSI" term_id="csi" >}} 驱动程序，因为它解决了 Flexvolumes 的限制。
```
Comment (chenrui333) [inline content/zh/docs/reference/glossary/flexvolume.md]:
```suggestion
如果可能的话，SIG Storage 建议实现 {{< glossary_tooltip text="CSI" term_id="csi" >}} 驱动程序，因为它解决了 Flexvolumes 的限制。
```

File: content/zh/docs/reference/glossary/flexvolume.md [hunk 2]
```diff
@@ -11,12 +26,26 @@ aka:
...
+
+ Flexvolume 是创建 out-of-tree 卷插件的一种接口。 {{< glossary_tooltip text="容器存储接口（CSI）" term_id="csi" >}} 是比 Flexvolume 更新的接口，它解决了 Flexvolume 的一些问题。
 
 <!--more--> 
 
+<!--
 Flexvolumes enable users to write their own drivers and add support for their volumes in Kubernetes. FlexVolume driver binaries and dependencies must be installed on host machines. This requires root access. The Storage SIG suggests implementing a {{< glossary_tooltip text="CSI" term_id="csi" >}} driver if possible since it addresses the limitations with Flexvolumes.
+-->
 
+Flexvolume 允许用户编写自己的驱动程序，并在 Kubernetes 中添加对卷的支持。
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/glossary/flexvolume.md]: Done！

File: content/zh/docs/reference/glossary/flexvolume.md (hunk 1 above)
Comment (AdamDang) [reply content/zh/docs/reference/glossary/flexvolume.md]: Done！

**Comment by AdamDang** [comment]
Thanks for reviewing！

**Pull request merged**